test/func/sproxyd_utils.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) 2014 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import with_statement
+import os
+import platform
+import subprocess32
+import urllib
+
+
+ring_driver_conf_tmpl = """"ring_driver:0": {
+        "by_path_service_id": "%(by_path_service_id)s",
+        "by_path_cos": 0,
+        "alias": "chord_path",
+        "bstraplist": "%(bootstraplist)s",
+        "by_path_enabled": true,
+        "deferred_deletes_enabled_by_policy": false,
```

Comment:
Ok, I do not remember from where I took the conf from. This could be a good reason to use the packaged config instead of hardcoding a config template .